README.md

```diff
@@ -45,26 +45,26 @@ Xournal++ features:
 - Fill shape functionality
 - PDF Export (with and without paper style)
 - PNG Export (with and without transparent background)
-- Allow to map different tools / colors etc. to stylus buttons / mouse buttons
+- Allowance to map different tools / colors etc. to stylus buttons / mouse buttons
 - Sidebar with Page Previews with advanced page sorting, PDF Bookmarks and Layers (can be individually hidden, editing layer can be selected)
-- enhanced support for image insertion
+- Enhanced support for image insertion
 - Eraser with multiple configurations
 - Significantly reduced memory usage and code to detect memory leaks compared to Xournal
-- LaTeX support (requires a working LaTeX install)
+- LaTeX support (requires a working LaTeX installation)
 - bug reporting, auto-save, and auto backup tools
-- Customizable toolbar, with multiple configurations, e.g. to optimize toolbar for portrait / landscape
+- Customizable toolbar with multiple configurations, e.g. to optimize toolbar for portrait / landscape
 - Page Template definitions
 - Shape drawing (line, arrow, circle, rect, splines)
 - Shape resizing and rotation
 - Rotation snapping every 15 degrees
 - Rect snapping to grid
 - Audio recording and playback alongside with handwritten notes
-- Multi Language Support, Like English, German (Deutsch), Italian (Italiano)...
+- Multi Language Support like English, German (Deutsch), Italian (Italiano)...
 - Plugins using Lua scripting
 
 ## Mobile & web app
 
-Since mid 2020, there is a Flutter-written mobile app for **Android**, **Chrome OS** and **iOS** (in coming) as well as a **web app** available. Even though it is not perfectly stable nor every of Xournal++'s features is supported yet, you may check it out and open your Xournal++ notebooks on your mobile devices. You can get in touch in it's [separate repository on GitLab](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile).
+Since mid 2020, there is a Flutter-written mobile app for **Android**, **Chrome OS** and **iOS** (coming soon) as well as a **web app** available. Even though it is not perfectly stable nor each of Xournal++'s features is supported yet, you may check it out and open your Xournal++ notebooks on your mobile devices. You can get in touch in it's [separate repository on GitLab](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile).
```

Comment:
```suggestion
Since mid 2020, there is a Flutter-written mobile app for **Android**, **Chrome OS** and **iOS** (coming soon) as well as a **web app** available. Even though it is not perfectly stable nor are all of Xournal++'s features fully supported yet, you may check it out and open your Xournal++ notebooks on your mobile devices. You can get in touch in it's [separate repository on GitLab](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile).
```